### DISCUSSION TITLE
ulimit not enough?

### DISCUSSION THREAD

**Discussion by xmflsct** [vercel/vercel#4149] [General] [answered]
Run into `too many open files` when building Gatsby site with merely 1500-ish pages.

See https://github.com/gatsbyjs/gatsby/issues/23463

**Comment by coetry** [comment, marked as answer]
Hi @xmflsct can you open up a ticket with us by emailing support@vercel.com so we can track your issue and look into the specifics of your project?

**Reply by xmflsct** [reply]
Thanks, I have reached out! And I also found a solution https://github.com/gatsbyjs/gatsby/issues/23463#issuecomment-619349042.
